lnrpc/stateservice.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+package lnrpc;
+
+option go_package = "github.com/lightningnetwork/lnd/lnrpc";
+
+/*
+ * Comments in this file will be directly parsed into the API
+ * Documentation as descriptions of the associated method, message, or field.
+ * These descriptions should go right above the definition of the object, and
+ * can be in either block or // comment format.
+ *
+ * An RPC method can be matched to an lncli command by placing a line in the
+ * beginning of the description in exactly the following format:
+ * lncli: `methodname`
+ *
+ * Failure to specify the exact name of the command will cause documentation
+ * generation to fail.
+ *
+ * More information on how exactly the gRPC documentation is generated from
+ * this proto file can be found here:
+ * https://github.com/lightninglabs/lightning-api
+ */
+
+// State service is a always running service that exposes the current state of
+// the wallet and RPC server.
+service State {
```

Comment:
Just some context: 
Technically we could make this a new RPC on one of the existing services, but I decided to make its own service since it simplifies the interceptor logic by always allowing calls to it regardless of wallet state.

If we were to add it to one of the existing services (`WalletUnlockerService` or `LightningService`) we would have to allow calls to that exact method regardless of wallet state. The service would also need to be aware of the wallet state somehow, while the way we do it now is by making the `Rpcinterceptor` itself provide this service.

Reply:
I think it's good to have this in its own file!